### PR TITLE
Create Partner-Billing-Guide

### PR DESCRIPTION
**Pull request by davidcardoza**
### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/318124
PROPOSAL: 

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
        - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

</details>

<details>
<summary>Mobile Web - Chrome</summary>

</details>

<details>
<summary>Mobile Web - Safari</summary>

</details>

<details>
<summary>Desktop</summary>

</details>

<details>
<summary>iOS</summary>

</details>

<details>
<summary>Android</summary>

</details>

### CODE REVIEW

**Review by rushatgabhane — CHANGES_REQUESTED** [review]
@davidcardoza please rename the file to add `.md` extension

`Partner-Billing-Guide` -> `Partner-Billing-Guide.md`

**Review by davidcardoza — COMMENTED** [review]
Looks good.

**Review by davidcardoza — COMMENTED** [review]
Looks good!

**Comment by davidcardoza** [comment]
@rushatgabhane bump on this

**Review by rushatgabhane — APPROVED** [review]
image

## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Partner Billing Guide
+description: Understand how Expensify bills partners and their clients
+---
+
+# Overview
+
+The ExpensifyApproved! Partner Program offers exclusive billing rates and features tailored for accountants to ensure seamless client management. If you are an accountant or consultant who recommends spend management solutions to your clients, becoming an ExpensifyApproved! Accountant may be a great certification for you. This guide will walk you through the unique billing perks available to ExpensifyApproved! partners, emphasizing the importance of understanding and managing client billing effectively. To learn what perks partners receive, check out the ExpensifyApproved! program details here.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:8]: > To learn what perks partners receive, check out the ExpensifyApproved! program details here.

Does `here` need to link to some page?

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+#  Get exclusive partner pricing
+
+All ExpensifyApproved! Partners are automatically eligible for a special rate of $9/seat monthly, without an annual commitment when they adopt the Expensify Card. This provides flexibility as active users can vary. Here are the specifics on pricing for our Approved! Partners’ clients:
+- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.s
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:13]:
```suggestion
- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+#  Get exclusive partner pricing
+
+All ExpensifyApproved! Partners are automatically eligible for a special rate of $9/seat monthly, without an annual commitment when they adopt the Expensify Card. This provides flexibility as active users can vary. Here are the specifics on pricing for our Approved! Partners’ clients:
+- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.s 
+- **Unbundled pricing (or pay-per-use pricing):** Clients not using the Expensify Card are billed at $18/seat monthly.
+- **No annual commitment:** Partners pay only for what they use, with no annual subscriptions required.
+- **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:16]:
```suggestion
- **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+# Understanding the billing process
+
+Expensify bills the owner of the expense policy for the activity on that policy. If accountants retain ownership of client policies, they receive the bill and can then re-bill their clients based on individual agreements.
+
+Each month, Expensify will send a consolidated bill detailing:
+- **Pay-per-use seats:** This is the number of active clients and their users for the month.
+- **Expensify Card discount**: This amount reflects how much spend is put on your card, which then determines the discount for that month.
+- **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:25]:
```suggestion
- **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+The Consolidated Domain Billing tool ensures that billing takes place under a single Expensify account associated with your firm. This eliminates the need to add multiple payment cards across various accounts to cover payments for multiple clients.
+
+## Maintaining a Console of all clients:
+
+If your firm wants to have a console view of all client policies and domains, you will want to create a single, centralized login to manage all client policies and domains, such as accounting@myfirm.com. 
+
+  1. Create a dedicated email address that will act as the universal policy owner, for example, accounting@myfirm.com.
+  2. Register this email with Expensify or your chosen platform and ensure it is verified and secured.
+  3. Within each client policy settings, add your centralized email (e.g. accounting@myfirm.com) as a policy admin.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:45]:
```suggestion
  3. Within each client workspace settings, add your centralized email (e.g. accounting@myfirm.com) as a workspace admin.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+### Activating Consolidated Domain Billing:
+  1. Claim and verify your firm’s domain.
+  2. Navigate to **Settings > Domains > [Domain Name] > Domain Admins** and set a **"Primary Domain Admin"** by using the drop down toggle to select an email address. 
+  3. Enable **Consolidated Domain Billing** in the same section.
+
+The Consolidated Domain Billing tool ensures that billing takes place under a single Expensify account associated with your firm. This eliminates the need to add multiple payment cards across various accounts to cover payments for multiple clients.
+
+## Maintaining a Console of all clients:
+
+If your firm wants to have a console view of all client policies and domains, you will want to create a single, centralized login to manage all client policies and domains, such as accounting@myfirm.com.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:41]:
```suggestion
If your firm wants to have a console view of all client workspaces and domains, you will want to create a single, centralized login to manage all client workspaces and domains, such as accounting@myfirm.com. 
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  2. Navigate to **Settings > Domains > [Domain Name] > Domain Admins** and set a **"Primary Domain Admin"** by using the drop down toggle to select an email address. 
+  3. Enable **Consolidated Domain Billing** in the same section.
+
+The Consolidated Domain Billing tool ensures that billing takes place under a single Expensify account associated with your firm. This eliminates the need to add multiple payment cards across various accounts to cover payments for multiple clients.
+
+## Maintaining a Console of all clients:
+
+If your firm wants to have a console view of all client policies and domains, you will want to create a single, centralized login to manage all client policies and domains, such as accounting@myfirm.com. 
+
+  1. Create a dedicated email address that will act as the universal policy owner, for example, accounting@myfirm.com.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:43]:
```suggestion
  1. Create a dedicated email address that will act as the universal workspsce owner, for example, accounting@myfirm.com.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  2. Register this email with Expensify or your chosen platform and ensure it is verified and secured.
+  3. Within each client policy settings, add your centralized email (e.g. accounting@myfirm.com) as a policy admin.
+  4. Do the same with each client domain.
+
+
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:48]:
```suggestion
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+# FAQ
+
+**Do I automatically get the special billing rate as an ExpensifyApproved! Partner?**
+- Yes, when you join the ExpensifyApproved! program, you will automatically get the special billing rate. To join the ExpensifyApproved! Program, you need to enroll in ExpensifyApproved! University. 
+
+**How can I check my billing details?**
+- To check your billing details, be on the lookout for your detailed billing statement sent out at the end of each month. 
+
+**Can I pass the bill to my clients?**
+- Yes, you can pass the bill on to your clients. If you retain ownership of client policies in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:77]:
```suggestion
- Yes, you can pass the bill on to your clients. If you retain ownership of client workspaces in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing. 
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+- To check your billing details, be on the lookout for your detailed billing statement sent out at the end of each month. 
+
+**Can I pass the bill to my clients?**
+- Yes, you can pass the bill on to your clients. If you retain ownership of client policies in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing. 
+
+**What if I don't want to use the Expensify Card?**
+- If you prefer not to use the Expensify Card, your clients will be billed at $18/seat monthly.
+
+**Why use client IDs?**
+- Client IDs provide a streamlined method to identify and manage policies, especially beneficial when a client has multiple policies.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:83]:
```suggestion
- Client IDs provide a streamlined method to identify and manage workspaces, especially beneficial when a client has multiple workspaces.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+# Understanding the billing process
+
+Expensify bills the owner of the expense policy for the activity on that policy. If accountants retain ownership of client policies, they receive the bill and can then re-bill their clients based on individual agreements.
+
+Each month, Expensify will send a consolidated bill detailing:
+- **Pay-per-use seats:** This is the number of active clients and their users for the month.
+- **Expensify Card discount**: This amount reflects how much spend is put on your card, which then determines the discount for that month.
+- **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats
+- **Policy list:** This is an overview of all client policies with their respective active seats.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:26]:
```suggestion
- **Workspace list:** This is an overview of all client workspaces with their respective active seats.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+## Consolidated Domain Billing
+
+If your firm wishes to consolidate all Expensify billing to a single account, the Consolidated Domain Billing feature is your go-to tool. It centralizes payment for all group policies owned by any domain member of your firm.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:30]:
```suggestion
If your firm wishes to consolidate all Expensify billing to a single account, the Consolidated Domain Billing feature is your go-to tool. It centralizes payment for all group workspaces owned by any domain member of your firm.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md (hunk 7 above)
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:43]:
```suggestion
  1. Create a dedicated email address that will act as the universal workspace owner, for example, accounting@myfirm.com.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.s 
+- **Unbundled pricing (or pay-per-use pricing):** Clients not using the Expensify Card are billed at $18/seat monthly.
+- **No annual commitment:** Partners pay only for what they use, with no annual subscriptions required.
+- **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill
+
+# Understanding the billing process
+
+Expensify bills the owner of the expense policy for the activity on that policy. If accountants retain ownership of client policies, they receive the bill and can then re-bill their clients based on individual agreements.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:20]:
```suggestion
Expensify bills the owner of the expense workspace for the activity on that workspace. If accountants retain ownership of client workspaces, they receive the bill and can then re-bill their clients based on individual agreements.
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+  1. Log in to your account: Ensure you’re using an Approved! accountant account.
+  2. Navigate to the desired Workspace: Go to **Settings > Workspaces > [Workspace Name] > Overview**.
+  3. Input the identifier: Here, you can input an alphanumeric unique identifier for each client workspace.
+**Note:** If a client has multiple workspaces, ensure each workspace has a consistent client ID.
+
+# How to access and download billing receipts
+Accessing Billing **Settings: Go to Settings > Your Account > Payments > Billing History.**
+Download the Receipt: Click on **"Download Receipt CSV".**
+
+
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:63]:
```suggestion
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+# How to assign a client ID to a workspace
+  1. Log in to your account: Ensure you’re using an Approved! accountant account.
+  2. Navigate to the desired Workspace: Go to **Settings > Workspaces > [Workspace Name] > Overview**.
+  3. Input the identifier: Here, you can input an alphanumeric unique identifier for each client workspace.
+**Note:** If a client has multiple workspaces, ensure each workspace has a consistent client ID.
+
+# How to access and download billing receipts
+Accessing Billing **Settings: Go to Settings > Your Account > Payments > Billing History.**
+Download the Receipt: Click on **"Download Receipt CSV".**
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:61]:
```suggestion
- Accessing Billing: **Settings: Go to Settings > Your Account > Payments > Billing History.**
- Download the Receipt: Click on **"Download Receipt CSV".**
```

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+
+
+# Deep Dive
+Using client IDs for all Workspaces: It's beneficial to use client IDs for all Workspaces to ensure each one is easily recognizable.
+Benefits of itemized billing receipts: Employing client IDs offers itemized billing by client, with each itemization detailing unique active users.
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:66]:
```suggestion
- Using client IDs for all Workspaces: It's beneficial to use client IDs for all Workspaces to ensure each one is easily recognizable.
- Benefits of itemized billing receipts: Employing client IDs offers itemized billing by client, with each itemization detailing unique active users.
```

**Comment by MonilBhavsar** [comment]
@davidcardoza @rushatgabhane 👋

**Comment by MonilBhavsar** [comment]
Some other improvements are left. @davidcardoza if you agree with suggestions, you can include them in a batch and commit at last. There's a guide here https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by davidcardoza** [comment]
Committed all suggestions

**Comment by MonilBhavsar** [comment]
Looks like few are still remaining

**Comment by davidcardoza** [comment]
Should be good here @MonilBhavsar

**Comment by MonilBhavsar** [comment]
Sorry, but some are still missing. They are hidden under this view as below

https://github.com/Expensify/App/assets/32012005/ac5f82a9-749e-4e63-9267-c90c0d4da80f

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md (hunk 8 above)
Comment (rushatgabhane) [reply docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:48]: it doesn't affect the way article is rendered. So it's ok to not make this change

**Comment by davidcardoza** [comment]
@MonilBhavsar All should be good here, apologies for the oversight on the additional commits.

File: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+**What if I don't want to use the Expensify Card?**
+- If you prefer not to use the Expensify Card, your clients will be billed at $18/seat monthly.
+
+**Why use client IDs?**
+- Client IDs provide a streamlined method to identify and manage workspaces, especially beneficial when a client has multiple workspaces.
+
+**Do I need a client ID for each Workspace?**
+- Yes, if you want to ensure seamless identification and billing processes.
+
+** Where can I see the Billing Receipts?**
```
Comment (MonilBhavsar) [inline docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md:86]:
```suggestion
**Where can I see the Billing Receipts?**
```

**Review by MonilBhavsar — COMMENTED** [review]
Thanks, no worries. A last missing thing that I notice we can fix before merging

**Review by MonilBhavsar — APPROVED** [review]
Thanks!

**Pull request merged**